core/src/main/java/io/spine/core/CommandEnvelope.java

```diff
@@ -113,13 +114,18 @@ public ActorContext getActorContext() {
     }
 
     /**
-     * Sets the context of the enclosed command as the origin for the event being built.
+     * Passes data from the enclosed command to the event context being built.
      *
-     * @param builder event context builder into which set the event origin context
+     * <p>In particular it sets the root command identifier and the event context origin. 
```

Comment:
Please have an unordered list in HTML. See other `<li>` examples in Spine for reference.